amplify/auth/resource.ts

```diff
@@ -41,5 +65,13 @@ export const auth = defineAuth({
     //   mutable: true,
     //   required: false,
     // },
+    givenName: {
+      mutable: true,
+      required: true,
+    },
```

Comment:
I think we can remove these since we are using DynamoDB to store these attributes